pycloudflare/models.py

```diff
@@ -7,31 +8,40 @@
 
 class User(object):
     def __init__(self, email, api_key):
-        self.api_key = api_key
         self.email = email
-        self.service = self.get_service(email, api_key)
+        self.api_key = api_key
```

Comment:
Changing it won't affect anything, so I'd drop it. It's not a very useful part of the interface.